libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -305,6 +304,11 @@ AP_BattMonitor::init()
                 drivers[instance] = new AP_BattMonitor_SMBus_NeoDesign(*this, state[instance], _params[instance]);
                 break;
 #endif
+            case Type::ACS37800_I2C:
+#if AP_BATTERY_ACS37800_ENABLED
+                drivers[instance] = new AP_BattMonitor_ACS37800(*this, state[instance], _params[instance]);
+#endif
+                break;
```

Comment:
```suggestion
#if AP_BATTERY_ACS37800_ENABLED
            case Type::ACS37800_I2C:
                drivers[instance] = new AP_BattMonitor_ACS37800(*this, state[instance], _params[instance]);
                break;
#endif
```

Reply:
Right, it was a bit confusing since the bebop just below has the other approach.